packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -96,40 +97,37 @@ import TileBase from './TileBase/index';
 
 interface ExportResultAsCSVModalProps {
     projectUuid: string;
-    savedChart: SavedChart;
+    chartUuid: string;
+    dashboardFilters?: DashboardFilters;
+    tileUuid?: string;
+    // Csv properties
     rows: ApiChartAndResults['rows'];
     onClose: () => void;
     onConfirm: () => void;
 }
 
 const ExportResultAsCSVModal: FC<ExportResultAsCSVModalProps> = ({
-    savedChart,
+    projectUuid,
+    chartUuid,
+    dashboardFilters,
+    tileUuid,
     rows,
     onClose,
     onConfirm,
 }) => {
-    const getCsvLink = async (limit: number | null, onlyRaw: boolean) => {
-        return downloadCsv({
-            projectUuid: savedChart.projectUuid,
-            tableId: savedChart.tableName,
-            query: savedChart.metricQuery,
-            csvLimit: limit,
-            onlyRaw: onlyRaw,
-            columnOrder: savedChart.tableConfig.columnOrder,
-            showTableNames: isTableChartConfig(savedChart.chartConfig.config)
-                ? savedChart.chartConfig.config.showTableNames ?? false
-                : true,
-            customLabels: getCustomLabelsFromTableConfig(
-                savedChart.chartConfig.config,
-            ),
-            hiddenFields: getHiddenTableFields(savedChart.chartConfig),
-            chartName: savedChart.name,
```

Comment:
struggling to follow if these (the showTableNames, customLabels, hiddenFields) have been done somewhere else or if it's handled already on the backend?

Reply:
Fixed, I copied this config into the backend